src/server-entry.js

```diff
@@ -182,6 +184,30 @@ export default context => {
 					logFormatter(`data pre-fetch: ${Date.now() - s}ms`);
 					sp = new Date();
 				}
+				const transactionId = router.currentRoute.query?.kiva_transaction_id
```

Comment:
I think this code to get the data should be in a function in `optimizelyUserMetrics.js`. Then server-entry can call it:
```js
import { buildUserDataGlobal } from '@/util/optimizelyUserMetrics';

...

__OPTIMIZELY_DATA__: buildUserDataGlobal(router, apolloClient)
```